lightning/src/util/events.rs

```diff
@@ -975,13 +1000,14 @@ impl Writeable for Event {
 				error_data.write(writer)?;
 				write_tlv_fields!(writer, {
 					(0, payment_hash, required),
-					(1, network_update, option),
+					(1, None::<NetworkUpdate>, option), // network_update in LDK versions prior to 0.0.114
```

Comment:
@TheBlueMatt it would be nicer backwards compat-wise to continue to write this field, and write the rest of the `Failure` data as a separate field later. It'll just mean that `Failure` can't be properly serialized on its own (which seems unlikely anyway). The current way seems a bit cleaner, but I'm guessing you'd prefer the way with more backwards compat? Let me know your thoughts

Reply:
Discussed offline, seems OK for old versions to read network update as `None`